src/components/SelectInstitution.tsx

```diff
@@ -96,6 +77,14 @@ const SelectInstitution: FC<SelectInstitutionProps> = ({ onSubmit, onClose }) =>
               <Autocomplete
                 options={institutions}
                 getOptionLabel={(option: RecursiveInstitutionUnit) => option.name}
+                filterOptions={(options: RecursiveInstitutionUnit[], state: any) => {
```

Comment:
state er for en eller annen grunn `object`(https://material-ui.com/api/autocomplete/), men hvis jeg bruker det klager den på at `inputValue` ikke finnes på `object`

Reply:
kan du ikke bruke typen fra Autocomplete.d.ts da?
```
export interface RenderOptionState {
  inputValue: string;
  selected: boolean;
}
```

Reply:
```
Type '(options: RecursiveInstitutionUnit[], state: RenderOptionState) => RecursiveInstitutionUnit[]' is not assignable to type '(options: RecursiveInstitutionUnit[], state: FilterOptionsState<RecursiveInstitutionUnit>) => RecursiveInstitutionUnit[]'.
```
Ser ut til at det virker med `state: FilterOptionsState<RecursiveInstitutionUnit>` :o
... men den kan bare importeres fra /useAutocomplete da

Reply:
weird. MUI er ikke så gode på typing altså

Reply:
kan fort være siden de Autocomplete fortsatt er i /lab